server/src/main/java/io/crate/replication/logical/LogicalReplicationService.java

```diff
@@ -218,6 +226,7 @@ private Runnable withRetry(Runnable runnable) {
     @Override
     public void close() throws IOException {
         metadataTracker.close();
+        // shardReplicationService.close();
```

Comment:
Initially did it inspired by https://github.com/crate/crate/commit/01d8da73d38d7c4507d4b0b7fb9c86b2e45d9096
but now think don't need it since we close trackers on Shard close/removal https://github.com/crate/crate/blob/logical-replication/server/src/main/java/io/crate/replication/logical/ShardReplicationService.java#L154

Reply:
Yes not needed.

Reply:
Actually, shards won't be closed/removed on node shutdown, so closing the `ShardReplicationService` should be indeed be done. But I suggest to do that here https://github.com/crate/crate/blob/logical-replication/server/src/main/java/org/elasticsearch/node/Node.java#L984 instead.